server/src/main/java/fr/uga/l3miage/tp1/exo2/models/User.java

```diff
@@ -0,0 +1,22 @@
+package fr.uga.l3miage.tp1.exo2.models;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import java.time.OffsetDateTime;
+
+@Entity
+@Table(name="user_entity")
+public class User {
```

Comment:
Le nom de tes entités doivent toujours avoir leur nom + "Entity" : 
```suggestion
public class UserEntity {
```